### PR TITLE
Ei 1910 reinstate payments

### PR DESCRIPTION
**Pull request by David-B-Read**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/ReinstatedPendingPaymentAudit.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using SFA.DAS.EmployerIncentives.Abstractions.Domain;
+
+namespace SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.ValueTypes
+{
+    public class ReinstatedPendingPaymentAudit : ValueObject
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/ReinstatedPendingPaymentAudit.cs:7]: This shouldn't be a domain object.  When a pending payment is reinstated you get back a PendingPayment object not a ReinstatedPendingPaymentAudit object.

**Review by Stephek — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/ReinstatedPendingPaymentAudit.cs (hunk 1 above)
Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/ReinstatedPendingPaymentAudit.cs:7]: This is used by the audit repository, and follows the same structure as the audit repository for the employment checks - EmploymentCheckRequestAudit is also a value type, which is mapped to the EF model for persistence

File: src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/IApprenticeshipIncentiveArchiveRepository.cs [hunk 2]
```diff
@@ -8,5 +9,6 @@ public interface IApprenticeshipIncentiveArchiveRepository
         Task Archive(PendingPaymentModel pendingPaymentModel);
         Task Archive(PaymentModel paymentModel);
         Task Archive(EmploymentCheckModel employmentCheckModel);
+        Task<PendingPaymentModel> Get(Guid pendingPaymentId);
```
[the remark this answers is not in the document]
Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/IApprenticeshipIncentiveArchiveRepository.cs]: Renamed method

File: src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/SFA.DAS.EmployerIncentives.Api.AcceptanceTests.csproj [hunk 3]
```diff
@@ -10,6 +10,7 @@
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\LegalEntityUpdated.feature.cs" />
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\PausePayments - Copy.feature.cs" />
     <SpecFlowObsoleteCodeBehindFiles Remove="Features\RecalculateEarnings.feature.cs" />
+    <SpecFlowObsoleteCodeBehindFiles Remove="Features\RevertPayments - Copy.feature.cs" />
```
[the remark this answers is not in the document]
Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/SFA.DAS.EmployerIncentives.Api.AcceptanceTests.csproj]: Removed

File: src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/ReinstatedPendingPaymentAudit.cs (hunk 1 above)
Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/ReinstatedPendingPaymentAudit.cs:7]: OK

**Review by Stephek — APPROVED** [review]

**Pull request merged**